traveler-swift-core/TravelerKit/Models/BookingItemDetails.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  BookingItemDetails.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-10-04.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// The detailed information of a bookable `ProductItem`
+public struct BookingItemDetails: CatalogItemDetails, Product, Decodable {
+    /// Identifier
+    public let id: String
+    /// Price
+    public let price: Price
+    /// Type
+    public let type: ProductType
+    /// Title
+    public let title: String
+    /// Description
+    public let description: String?
+    /// An array of URLs of images
+    public let imageUrls: [URL]
+    /// Attributes
+    public let information: [Attribute]?
+    /// Vendor's contact information
+    public let contact: ContactInfo?
+    /// An array of locations
+    public let locations: [Location]
+    /// Product supplier
+    public let supplier: Supplier
+    /// Disclaimer
+    public let disclaimer: String?
+    /// Terms and conditions
+    public let termsAndConditions: String?
+    /// Indicating if its wishlisted
+    public let isWishlisted: Bool?
+    /// Google translate attribution 
+    public let translateAttribution: GoogleTranslateAttribution
```

Comment:
Why is the model `Google....`?

Model should be generic and have no mention of google, if the API's response also has this type of specific naming it should be brought up and changed